.build/docker/run-tests.sh

```diff
@@ -27,7 +27,7 @@
 # help
 if [ "$#" -lt 1 ] || [ "$#" -gt 3 ] || [ "$1" == "-h" ]; then
     echo ""
-    echo "Usage: run-tests.sh target [split_chunk|test_regexp] [java_version]"
+    echo "Usage: run-tests.sh test_type [split_chunk|test_regexp] [java_version]"
```

Comment:
would be nice for the help message to include the valid test types

Reply:
yes, i've been thinking about this, but don't have a clear way. lhf to leave.